src/utils/validator.js

```diff
@@ -18,4 +18,34 @@ export default class Validator {
     });
     return errors;
   }
+
+  /**
+   * @description checks if an email syntax is right or wrong
+   * @param { String } email
+   * @returns { Boolean } Boolean
+   */
+  static isEmailValid(email) {
+    const emailRegex = /^([a-z_.!@#$%^&*0-9]{3,25})@([a-z]{3,20})\.([a-z]){2,7}(\.[a-z]{2,5})?$/i;
```

Comment:
Hope you are able to explain this because the last time I used regex, I got a warning.

Reply:
Loool!!!!
I can try to explain that...
I used it because I needed to validate the email on the frontend without using any library

Reply:
I understand